spec/requests/items_spec.rb

```diff
@@ -94,36 +95,34 @@
 
   describe "POST api/v1/items" do
     context "when item post request attributes are valid" do
+      valid_attributes = {name: "Asparagus", sellIn: 10, quality: 25, description: "stick-like green vegetable"}
+
       it "creates a new item in the database" do
-        valid_attributes = {name: "Asparagus", sellIn: 10, quality: 25}
 
         expect { post "/api/v1/items", params: valid_attributes }.to change(Item, :count).by(+1)
       end
 
       it "returns status code 201" do
-        valid_attributes = {name: "Asparagus", sellIn: 10, quality: 25}
         post "/api/v1/items", params: valid_attributes
 
         expect(response).to have_http_status(201)
       end
     end
 
-    context "wwhen item post request attributes are invalid" do
+    context "when item post request attributes are invalid" do
+      invalid_attributes = {title: "Lorem Ipsum"}
       it "does not create a new item in the database" do
-        invalid_attributes = {title: "Lorem Ipsum"}
```

Comment:
Yes this is dry, but the tests should be "wet" for readability so when someone reads each "it" block in isolation they don't have to jump to find what "invalid params" is.

Reply:
fixed